user-access-event-handlers/src/main/java/no/unit/nva/useraccess/events/service/UserMigrationServiceImpl.java

```diff
@@ -47,11 +51,15 @@ private void removeOldPatternOrganizationIds(String username) {
             var systemControlNumber = authority.get().getSystemControlNumber();
             var organizationIds = authority.get().getOrganizationIds();
             organizationIds.stream()
-                .filter(ViewingScope::isNotValidOrganizationId)
+                .filter(this::isUriToCristinApi)
                 .forEach(uri -> deleteFromAuthority(systemControlNumber, uri));
         }
     }
 
+    private boolean isUriToCristinApi(URI organizationId) {
```

Comment:
private boolean isUriToCristinApi(URI organizationId) {
        return CRISTIN_API_URI.equals(organizationId.getHost());
    }

Reply:
I suspect we will need this to be an env variable and therefore I find it much easier if we give only the hosts, for all external services and not URIs. If we give URIs, then we have to check, is it https, is it this, is it that.